src/logic/addressBook/model/addressBook.ts

```diff
@@ -1,17 +1,28 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+import { getNetworkId } from 'src/config'
+
+export const ADDRESS_BOOK_DEFAULT_NAME = 'UNKNOWN'
+
 export type AddressBookEntry = {
-  address: string
-  name: string
+  address: string // the contact address
+  name: string // human-readable name
+  chainId: ETHEREUM_NETWORK // see https://chainid.network
 }
 
+const networkId = getNetworkId()
+
 export const makeAddressBookEntry = ({
   address = '',
   name = '',
+  chainId = networkId,
```

Comment:
does it make sense to receive this as an argument? I can not imagine setting a different value than the current network

Reply:
yes, I agree.

Currently, there's no implementation that's using it. It's always using the default value.

But I didn't use much time to think about it, maybe there's a case we're missing and it won't hurt having it by the time being.